src/Umbraco.Web/PropertyEditors/ValueConverters/MediaPickerValueConverter.cs

```diff
@@ -31,15 +39,19 @@ public override Type GetPropertyValueType(PublishedPropertyType propertyType)
             var isMultiple = IsMultipleDataType(propertyType.DataType);
             var isOnlyImages = IsOnlyImagesDataType(propertyType.DataType);
 
-            // hard-coding "image" here but that's how it works at UI level too
-
             return isMultiple
-                ? (isOnlyImages ? typeof(IEnumerable<>).MakeGenericType(ModelType.For("image")) : typeof(IEnumerable<IPublishedContent>))
-                : (isOnlyImages ? ModelType.For("image") : typeof(IPublishedContent));
+                ? isOnlyImages
+                    ? typeof(IEnumerable<>).MakeGenericType(ModelType.For(ImageTypeAlias))
+                    : typeof(IEnumerable<IPublishedContent>)
+                : isOnlyImages
+                    ? ModelType.For(ImageTypeAlias)
+                    : typeof(IPublishedContent);
         }
 
         public override PropertyCacheLevel GetPropertyCacheLevel(PublishedPropertyType propertyType)
-            => PropertyCacheLevel.Snapshot;
+        {
```

Comment:
why?

Reply:
Just, did a file format in Rider. Maybe I had to tweak the settings. Right now it uses default settings that should be based on what is used most in the code base. Can tweak this.